sst/services/functions/fanout.ts

```diff
@@ -1,15 +1,16 @@
+import { Topic } from '@serverless-stack/node/topic';
 import aws from 'aws-sdk';
-import  { DynamoDBStreamHandler } from 'aws-lambda';
+import { DynamoDBStreamHandler } from 'aws-lambda';
 
 const sns = new aws.SNS();
 
 export const handler: DynamoDBStreamHandler = async (event) => {
   const items = event.Records.map((record) => record.dynamodb && record.dynamodb.NewImage && aws.DynamoDB.Converter.unmarshall(record.dynamodb.NewImage) || undefined).filter((item) => item);
   console.log('items to be published', items);
-  
+
   for (const item of items) {
     const params = {
-      TopicArn: process.env.TOPIC_ARN,
+      TopicArn: Topic.topic.topicArn,
```

Comment:
Since `topic` was bound to this function, we can reference `topic.TopicArn` in a type safe way.